### PR TITLE
Move pylint settings from .toml files to .pylintrc

### PR DESCRIPTION
**Pull request by mzappitello**
Unify our pylint settings across both of our python modules by putting
them in a single rc file rather than configuring them in seperate dot
toml files. It will also better work with vs code, which has issues with
multiple pylint settings inside of one project.

Update ci yaml files that are used by github actions to use the rc file.

Asana Task: <asana_ticket_url>

### CODE REVIEW

File: .github/workflows/ingestion-ci.yaml [hunk 1]
```diff
@@ -56,4 +56,4 @@ jobs:
       - uses: ./.github/actions/poetry_deps
         with:
           poetry-dir: py_gtfs_rt_ingestion
-      - run: poetry run pylint *.py lib/ tests/
+      - run: poetry run pylint --rcfile ../.pylintrc *.py lib/ tests/
```
Comment (mzappitello) [inline .github/workflows/ingestion-ci.yaml:59]: use the pylintrc file for linting in ci workflows.

File: .pylintrc [hunk 2]
```diff
@@ -0,0 +1,627 @@
+[MAIN]
```
Comment (mzappitello) [inline .pylintrc:1]: i generated this file using pylint and then added in the custom settings we had setup in our toml files.

File: performance_manager/pyproject.toml [hunk 3]
```diff
@@ -2,8 +2,11 @@
 name = "performance_manager"
 version = "0.1.0"
 description = "Compare Static Schedule and Real Time GTFS"
-authors = ["MBTA CTD <developer@mbta.com>",
-           "Mike Zappitello <mzappitello@mbta.com>"]
+authors = [
+  "MBTA CTD <developer@mbta.com>",
+  "Ryan Rymarczyk <rrymarczyk@mbta.com>",
+  "Mike Zappitello <mzappitello@mbta.com>",
+]
```
Comment (mzappitello) [inline performance_manager/pyproject.toml:9]: better formatting and added ryans email.

File: performance_manager/pyproject.toml [hunk 4]
```diff
@@ -31,32 +34,11 @@ target-version = ['py39']
...
-  # we're logging everything so its "ok"
-  "lost-exception"
-]
-good-names = ["i", "s"]
-max-line-length = 80
-min-similarity-lines = 10
-# ignore session maker as it gives pylint fits
-# https://github.com/PyCQA/pylint/issues/7090
-ignored-classes = ['sqlalchemy.orm.session.sessionmaker' ]
-
```
Comment (mzappitello) [inline performance_manager/pyproject.toml:51]: moved to pylintrc

File: py_gtfs_rt_ingestion/pyproject.toml [hunk 5]
```diff
@@ -33,23 +36,6 @@ warn_unused_ignores = true
...
-  "line-too-long",
-  # for some reason Iterable[type] is triggering this error on github
-  "unsubscriptable-object",
-  # Converter abstract base class only has one common function
-  "too-few-public-methods"
-]
-good-names = ["e", "i"]
-max-line-length = 80
-min-similarity-lines = 10
-
```
Comment (mzappitello) [inline py_gtfs_rt_ingestion/pyproject.toml:52]: moved to pylintrc

**Comment by rymarczy** [comment]
LGTM!

**Pull request merged**